config/install/user.role.manage_content.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
     - captcha
     - contextual
     - dropzonejs
+    - editoria11y
```

Comment:
Do these module addition need to be included in the update function?

Reply:
it builds the dependency tree for Drupal and no they could be left off. This just ensures that the module is installed before trying to import this config

Reply:
Also no they do not need to be included in this update as the module is already required for all sites